modules/developer-cli-odo-installing-odo-on-linux-on-IBM-Power.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/installing-odo.adoc
+
+[id="installing-odo-on-linux-on-IBM-Power"]
+
+= Installing {odo-title} on Linux on IBM Power
+
+== Binary installation
```

Comment:
Please also ensure that the ID includes reference to the context (`[id="module-name_{context}"]`).

Reply:
On further review and discussion within the peer review team, this subheaded section could potentially be split out into an individual module. This would help if another team wanted to reuse just this one section in future.

Reply:
The DevTools team (@boczkowska) will fix the individual modules via a separate PR please.